### PR TITLE
ISBNdb Plugin

### PR DESCRIPTION
**Pull request by K0HAX**
I created a plugin to allow users to type ".isbn <Book ISBN>" and return information about the book from http://isbndb.com.

### CODE REVIEW

File: plugins/isbn.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import re
+import json, requests
```
Comment (edwardslabs) [inline plugins/isbn.py]: You shouldn't need to import json as requests has a built in json method.

File: plugins/isbn.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+    isbndb_key = bot.config.get("api_keys", {}).get("isbndb_dev_key", None)
+
+def getISBN(book):
+    try:
+        url = "http://isbndb.com/api/v2/json/{}/book/".format(isbndb_key)
+        url = url + book
+        #print(url)
+        resp = requests.get(url=url)
+        data = json.loads(resp.text)
```
Comment (edwardslabs) [inline plugins/isbn.py]: this could be changed to 
```
data = resp.json()
```

**Comment by knight-ryu12** [comment]
yup, But It still requires json module for getting dict from json str.


2017/04/18 午前7:10 "Andy Edwards" <notifications@github.com>:

**Comment by daboross** [comment]
Requests has that built in though, with the `response.json()` method - no need to depend directly on json?

**Comment by K0HAX** [comment]
I just pushed an update using ```response.json()```, I've tested that method and it worked.

**Pull request merged**